docs/departments/law-enforcement/standard-operating-procedures.mdx

```diff
@@ -112,22 +114,29 @@ Assistant Commissioner
 
 Trooper/Officer/Deputy
 
 Probationary
 
-#### 2.1.1 - The Ranking Structure
+The ranks in the COC signify a level of experience within the department. Following with this comes an authority in some areas. However, who you report to may not necessariliy be the next position up. 
+
+For example, a probationary will directly report to a Feild training officer. Even though any supervisor has the experience for this duty, you will report to the ones who are FTO certified. They hae gone through additional training and gained special access to help you out. Some supervisors may have other duties, hence why they have not opted into this program. Note: This does not mean FTOs have authority over other supervisors.
 
-The department of Law Enforecement will be considered a uniform department. The ranks of the police departments will be consistant through out. The rank you are given will be valid in all departments, this creats a flow of the three departments. If you apply for Law Enforcemt at Venture RP and you are accepeted, you will be granted the Probationary rank and access to all three departments in there respective Probationary positions. (Probatinary Trooper, Probationary Officer, Probationary Deputy) 
+Further explainations based on this topic will be answered in the next section.
 
-Probationary - as a Probationary you main goals should be to learn as much as possible about how the department runs as well as the expectations of professionalism at Venture RP's LEO Deaprtment. You should be maintaining a high level of repect for the Feild Training Officers (FTO) as they will be the ones to teach you how to be succsessful in Venture RP. You will be not allowed to patrol on your own, only with a certified FTO. You must complete a series of FTO ride alongs before you can be promoted to a stand alone rank (Trooper, Officer, Deputy).  
+#### 2.2 - Duties and responsibilities of the COC
 
+It is importnant to adhere to all responisibilites. Please note this list is not the limit of your duties only a profile of what is expected at VRP.
 
+#### 2.2.1 - Probationary 
 
+As a Probationary you main goals should be to learn as much as possible about how the department runs, as well as the expectations of professionalism at Venture RP's LEO Deaprtment. You should be maintaining a high level of repect for the Feild Training Officers (FTO) as they will be the ones to teach you how to be succsessful in the LEO Department and further, at Venture RP. 
 
+At this stage you will not be allowed to patrol on your own, only with a certified FTO. You must complete a series of FTO ride alongs before you can be promoted to a stand alone rank (Trooper, Officer, Deputy).  
 
+Based on your performance you will be given various opertunities to operate the police vehicle and other patrol duties while acompanied by an FTO. More info on your training will be provided in the training section of this SOP
 
 <h3 align="center">Section 3</h3>
```

Comment:
Should be in the format of the below example for **all** sections
```mdx
## Section 3 - General Ethics
```

This will improve page indexing. Should only be an `h2` with the double hashtag (`##`)